### ISSUE TITLE
Meta-package to install everything

### ISSUE THREAD

**Issue by lassik** [Unibeautify/unibeautify#187]
Would it be possible to make some kind of meta-package for npm so that users could install everything (the Unibeautify core, the CLI, and all known beautifiers) with a single command? I imagine lots of users would want this, as it's not that big of a deal even when pulling in all dependencies (prettydiff, etc.) People are used to 100 megabyte plus installs, and the convenience is well worth it.

This would also be a natural springboard for getting Unibeautify into OS package managers. The OS packages could be based on this metapackage (or some kind of [rollup](https://rollupjs.org/) version of same).

I was going over the install instructions for the Emacs package and realized that all the various npm modules seems quite complex for people coming from that background. The npm package syntax with all the `@` signs alone seems a little intimidating. A meta-package would really help with this.

**Comment by Glavin001** [comment]
Good idea. I think @szeck87 and I have discussed in the past a package like `unibeautify-packed` where it comes with everything all in one and then it is reused for VSCode, Atom, etc for easier installation.

**Comment by lassik** [comment]
That's great :) Does npm have a ready-made mechanism for making such a package and it's just a matter of putting in the effort, or is some investigation and experimentation required?

**Comment by Glavin001** [comment]
I am not aware of any ready-made mechanism from `npm` itself. Will have to create a npm package. Here's a repo for anyone interested in contributing: https://github.com/Unibeautify/packed
